patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch

```diff
@@ -0,0 +1,26 @@
+From 7dbf2689eb841c51dca4dad51b0941c06aa09e26 Mon Sep 17 00:00:00 2001
+From: Vadym Hlushko <vadymh@nvidia.com>
+Date: Mon, 11 Apr 2022 15:41:46 +0000
+Subject: [PATCH 3/3] psample: added define PSAMPLE_MD_EXTENDED_ATTR
+
+Signed-off-by: Vadym Hlushko <vadymh@nvidia.com>
+---
+ include/net/psample.h | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/include/net/psample.h b/include/net/psample.h
+index e328c5127..b7c79f634 100644
+--- a/include/net/psample.h
++++ b/include/net/psample.h
+@@ -14,6 +14,8 @@ struct psample_group {
+ 	struct rcu_head rcu;
+ };
+ 
++#define PSAMPLE_MD_EXTENDED_ATTR   1
```

Comment:
Where is this used?

Reply:
Seems like it was never used - deleted patch